intermediate/intermediate5/customer.py

```diff
@@ -0,0 +1,29 @@
+from product import Product
+class Customer:
+    def __init__(self, name):
+        self.name = name
+        self.shopping_list = []
+        self.total_purchase_amount = 0 
+
+    def add_product(self, product):
+        for existing_product in self.shopping_list:
```

Comment:
too long a name. unnecessary. I know it exists. use for product in self.shopping_list